site/docs/v1/tech/tutorials/integrate-ruby-rails-api.md

```diff
@@ -0,0 +1,193 @@
+```
+Then, you can run the setup script.
+
+{% include _callout-note.liquid content="The setup script is designed to run on a newly installed FusionAuth instance with only one user and no tenants other than `Default`. To follow this guide on a FusionAuth instance that does not meet these criteria, you may need to modify the above script. <br><br> Refer to the [Ruby client library](/docs/v1/tech/client-libraries/ruby) documentation for more information." %}
+
+This will create the FusionAuth configuration for your {{page.technology}} API.
+
+```shell
+fusionauth_api_key=YOUR_API_KEY_FROM_ABOVE ruby setup.rb
+```
+
+If you are using PowerShell, you will need to set the environment variable in a separate command before executing the script.
+
+```shell
+$env:fusionauth_api_key='YOUR_API_KEY_FROM_ABOVE'
+ruby setup.rb
+```
+
+If you want, you can http://localhost:9011[login to your instance] and examine the new API configuration the script created for you. 
+
+## Create Your {{page.technology}} API
+
+Now you are going to create a {{page.technology}} API. While this section builds a simple {{page.technology}} API, you can use the same configuration to build a more complex {{page.technology}} API.
+
+First, create the skeleton of the {{page.technology}} API. Rails has a nice generator to build this out.
+
+```shell
+rails new myapi --api && cd myapi
+```
+
+Now, create a new `Gemfile` file.
+
+```text
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-rails-api-guide/main/Gemfile %}
+```
+
+Then, install these new gems.
+
+```shell
+bundle install
+```
+
+Next, create a file called `.env.development` and insert the following into it.
+
+```ini
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-rails-api-guide/main/.env.development %}
+```
+
+You can now start writing the code for your Rails API. First, let's create a controller which gives back a JSON message. Create a new file called `app/controllers/messages_controller.rb`, then add the following code:
+
+```ruby
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-rails-api-guide/main/app/controllers/messages_controller.rb %}
+```
+
+This controller returns a JSON array with messages.
+
+Next, update the `config/routes.rb` file to look like this:
+
+```ruby
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-rails-api-guide/main/config/routes.rb %}
+```
+
+This tells {{page.technology}} to return the content generated by the `messages_controller.rb` file when the `/messages` path is set up.
+
+You can now start up your server. You should do it in a new terminal window so that you can continue to edit the {{page.technology}} code.
+
+```shell
+bundle e rails s -p 4001
+```
+
+And visit [http://localhost:4001/messages](http://localhost:4001/messages) and view the JSON.
```

Comment:
This renders an error on line 7 of include.
<img width="983" alt="Screenshot 2023-05-17 at 1 18 29 PM 1" src="https://github.com/FusionAuth/fusionauth-site/assets/11809481/c6a27a4e-baf4-4302-9c75-e8704fc01665">

Reply:
Thanks, resolved.